src/mad_spark_alt/cli.py

```
@@ -40,6 +41,58 @@
 console = Console()
 
 
+def _get_semantic_operator_status() -> str:
+    """Get status of semantic operators (ENABLED/DISABLED)."""
+    if LLMProvider.GOOGLE in llm_manager.providers:
+        return "Semantic operators: ENABLED"
+    else:
```

Ruff (0.12.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)